backup/setup/systems.json

```diff
@@ -18,8 +18,25 @@
 		"_id": {
 			"$oid": "0000d186816abba584714c92"
 		},
-		"type": "iserv",
+		"alias": "Zentrales IServ",
+		"type": "ldap",
 		"__v": 0,
+		"ldapConfig": {
+			"active": true,
+			"federalState": {
+				"$oid": "0000b186816abba584714c58"
+			},
+			"url": "ldap://sc-openldap-svc.sc-common.svc.cluster.local:389",
+			"rootPath": "dc=de,dc=example,dc=org",
+			"searchUser": "cn=admin,dc=example,dc=org",
+			"searchUserPassword": "U2FsdGVkX19xqHusWE7tJnRotOIykWITaT9kSUwEusM=",
+			"provider": "iserv-idm",
+			"providerOptions": {
+				"userAttributeNameMapping": {},
+				"roleAttributeNameMapping": {},
+				"classAttributeNameMapping": {}
+			}
```

Comment:
Rejected, don't add the stuff from the devcluster

Reply:
already removed